File: jvm/cucumber-blueprint-jvm-example/src/test/java/cucumber/blueprint/jvm/example/integration/cucumber/scan/ExampleRouteBuilderITest.java
package cucumber.blueprint.jvm.example.integration.cucumber.scan;

import cucumber.api.CucumberOptions;
import cucumber.api.junit.Cucumber;
import org.junit.runner.RunWith;

@RunWith(Cucumber.class)
@CucumberOptions(features = "classpath:features/exampleroute.feature", glue = {
        "cucumber.blueprint.jvm.example.integration.cucumber.step.exampleroute"}, strict = true)
public class ExampleRouteBuilderITest {
}

File: scala/cucumber-blueprint-scala-example/src/test/scala/cucumber/blueprint/jvm/example/integration/cucumber/scan/ExampleRouteBuilderITest.scala
package cucumber.blueprint.jvm.example.integration.cucumber.scan

import org.junit.runner.RunWith
import cucumber.api.CucumberOptions
import cucumber.api.junit.Cucumber

@RunWith(classOf[Cucumber])
@CucumberOptions(features = Array("classpath:features/exampleroute.feature"), glue = Array("cucumber.blueprint.jvm.example.integration.cucumber.step.exampleroute"), strict = true)
class ExampleRouteBuilderITest
